integration-tests/utils/integration-utils.js

```diff
@@ -273,8 +273,9 @@ const findInDataTableRow = (page, entityName, text) => {
   return findElement(page, elementInDataTableRow(entityName, text))
 }
 
-const findButtonInDialogByAriaLabel = (page, ariaLabelText) => {
-  return page.waitForXPath(`//*[@role="dialog" and @aria-hidden="false"]//*[@role="button" and contains(@aria-label,"${ariaLabelText}")]`,
+const findButtonInDialogByAriaLabel = (page, ariaLabelText, isDisabled = false) => {
```

Comment:
I don't love the lone boolean, especially because callers passing `true` means that it should be disabled, which feels backwards. What would you think about an options object?
```suggestion
const findButtonInDialogByAriaLabel = (page, ariaLabelText, { isDisabled = false } = {}) => {
```
Then calls would look like `findButtonInDailogByAriaLabel(page, 'Click Me', { isDisabled: true })`.

Reply:
I like this 👍🏻